pkg/core/keyutils.go

```
@@ -54,6 +54,8 @@ func GeneratePrivateKey(signatureAlg x509.SignatureAlgorithm) (keys any, err err
 		if err != nil {
 			return nil, err
 		}
+	default:
+		keys, err = rsa.GenerateKey(rand.Reader, 2048)
```

golangci-lint (1.64.8)

ineffectual assignment to err

(ineffassign)